### PR TITLE
Add AutoFFI 0.0.1-alpha

### PR DESCRIPTION
**Pull request by ghost**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].
<hr>
AutoFFI is a tool for autogenerating FFI bindings, currently in incubation phase. This commit adds version v0.0.1-alpha.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
There was a [pull request template](https://github.com/blog/2111-issue-and-pull-request-templates) when you submitted. It must be filled.

If you were not presented a template, you may have submitted via a CLI tool (likely [`hub`](https://hub.github.com/)). It that was the case, you may wish to see if the tool has a way to let you use the template (`hub` [is working on it](https://github.com/github/hub/issues/1118)).

In either case, please [refer to the template](https://raw.githubusercontent.com/caskroom/homebrew-cask/master/.github/PULL_REQUEST_TEMPLATE.md) and update your comment.

File: Casks/autoffi.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'autoffi' do
+  version '0.0.1'
+  sha256 'd17a2a70bfe28c9dff8338c6c5851e2acfaf10357e8595ff0e46cd72ad64e6dc'
+
+  url 'https://github.com/AutoFFI/AutoFFI/releases/download/v0.0.1-alpha/autoffi-v0.0.1-alpha-osx-sierra-10.12.tar.gz'
```
Comment (vitorgalvao) [inline Casks/autoffi.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

Comment (vitorgalvao) [inline Casks/autoffi.rb]: `url "https://github.com/AutoFFI/AutoFFI/releases/download/v#{version}-alpha/autoffi-v#{version}-alpha-osx-sierra-10.12.tar.gz"`

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
There are multiple problems with this submission, in addition to those I’ve pointed out. The others will be fixed by following the template, though.

**Comment by samvv** [comment]
Sorry about that, I was indeed using hub. I have corrected the mistakes. Thank you for your time and please let me know if there is anything else that needs changing.

**Comment by samvv** [comment]
**Update:** still something wrong, I'm investigating it.

File: Casks/autoffi.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask 'autoffi' do
+  version '0.0.1'
+  sha256 'd17a2a70bfe28c9dff8338c6c5851e2acfaf10357e8595ff0e46cd72ad64e6dc'
+
+  url 'https://github.com/AutoFFI/AutoFFI/releases/download/v#{version}-alpha/autoffi-v#{version}-alpha-osx-sierra-10.12.tar.gz'
```
Comment (vitorgalvao) [inline Casks/autoffi.rb]: You need double quotes, here.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/autoffi.rb (hunk 2 above)
Comment (samvv) [inline Casks/autoffi.rb]: Just found that out, thanks.

**Comment by samvv** [comment]
Just for the record: if I'm correct it should be good to go now.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**